test/integration/integration_test.go

```diff
@@ -150,11 +151,15 @@ func FeatureContext(s *godog.Suite) {
 		minishift.shouldHaveState)
 	s.Step(`^executing "minishift ([^"]*)"$`,
 		minishift.executingMinishiftCommand)
+	s.Step(`^executing »minishift ([^«]*)«$`,
```

Comment:
I think the problem is in this regexp '^executing "minishift ([^"]*)"$'. Matches are greedy, but '[^"]' forces it to stop after the first occurnace of a quote.

I think it should be just '^executing "minishift (.*)"$'. That should make the need to add another step obsolete.